engine/src/main/java/org/hibernate/search/query/dsl/impl/FacetRange.java

```diff
@@ -87,6 +87,20 @@ private String buildRangeString() {
 	}
 
 	@Override
+	public int hashCode() {
```

Comment:
Overriding hashCode() but not equals()? That seems not right. Is there a need for this (i.e. is it used in sets or maps as key or so)?